.github/workflows/sssd.yml

```diff
@@ -13,46 +13,21 @@ concurrency:
 # END OF COMMON SECTION
 
 jobs:
-  build_wolfssl:
-    name: Build wolfSSL
-    # Just to keep it the same as the testing target
-    runs-on: ubuntu-22.04
-    # This should be a safe limit for the tests to run.
-    timeout-minutes: 4
-    steps:
-      - name: Build wolfSSL
-        uses: wolfSSL/actions-build-autotools-project@v1
-        with:
-          path: wolfssl
-          configure: --enable-all CFLAGS=-DWOLFSSL_NO_ASN_STRICT
-          install: true
-          check: false
-
-      - name: tar build-dir
-        run: tar -zcf build-dir.tgz build-dir
-
-      - name: Upload built lib
-        uses: actions/upload-artifact@v4
-        with:
-          name: wolf-install-sssd
-          path: build-dir.tgz
-          retention-days: 5
-
   sssd_check:
```

Comment:
Why are we including building wolfSSL in the `sssd_check` job?

Reply:
I was just consolidating the job to be simpler. I didn't see a need to upload an artifact only to download it again. I could change the name of the stage if that's better.

Reply:
Agreed, but we're only testing a single version so it seemed superfluous. I'll revert this change.

Reply:
It decreases the workload on the person updating the support. I think its worth it to separate on the initial write.